app/controllers/samples_controller.rb

```diff
@@ -214,7 +214,7 @@ def bulk_process_csv
       CSV.open(csv_file.path) do |csv_stream|
         csv_stream.each do |row|
           sample_id, measured_signal = row[0], row[1]
-
+          next if sample_id == "sample_UUID" # skip header row
```

Comment:
It might be better to check whether it's an UUID string with a simple regexp.

Reply:
While in javascript I re-used 

`/^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi`

the `g` option didn't worked on ruby and I had to go with 

`/^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/i`

I get a `SyntaxError ((irb):11: unknown regexp option - g)` 

It should work well in any case, just mentioning it